docs/data/material/getting-started/templates/checkout/PaymentForm.js

```diff
@@ -14,12 +14,56 @@ import RadioGroup from '@mui/material/RadioGroup';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
 
+import { styled } from '@mui/material/styles';
+
 import AccountBalanceRoundedIcon from '@mui/icons-material/AccountBalanceRounded';
 import CreditCardRoundedIcon from '@mui/icons-material/CreditCardRounded';
 import SimCardRoundedIcon from '@mui/icons-material/SimCardRounded';
 import WarningRoundedIcon from '@mui/icons-material/WarningRounded';
 
-import { styled } from '@mui/system';
+const Card = styled(MuiCard)(({ theme }) => ({
+  border: '1px solid',
+  borderColor: theme.palette.divider,
+  width: '100%',
+  '&:hover': {
+    background:
+      theme.palette.mode === 'light'
+        ? 'linear-gradient(to bottom right, hsla(210, 100%, 97%, 0.5) 25%, hsla(210, 100%, 90%, 0.3) 100%)'
+        : 'linear-gradient(to right bottom, hsla(210, 100%, 12%, 0.2) 25%, hsla(210, 100%, 16%, 0.2) 100%)',
+    borderColor: theme.palette.mode === 'light' ? 'primary.light' : 'primary.dark',
+    boxShadow:
+      theme.palette.mode === 'light'
+        ? '0px 2px 8px hsla(0, 0%, 0%, 0.1)'
+        : '0px 1px 8px hsla(210, 100%, 25%, 0.5) ',
+  },
+  [theme.breakpoints.up('md')]: {
+    flexGrow: 1,
+    maxWidth: `calc(50% - ${theme.spacing(1)})`,
+  },
+}));
+
+const StyledBox = styled('div')(({ theme }) => ({
```

Comment:
Box as MuiBox here, too

Reply:
I was wondering what the best approach would be in this case. In practice, it is a "StyledDiv", and not actually a `Box`

Reply:
Ah, I didn't realize this was a plain div styled. I think you can give it a more meaningful name, then, like "PaymentContainer", or something along these lines.